src/pkg/clouds/aws/ecs/logs.go

```diff
@@ -313,13 +312,13 @@ func (c *collectionStream) addAndStart(s EventStream, since time.Time, lgi LogGr
 		defer c.wg.Done()
 		if !since.IsZero() {
 			// Query the logs between the start time and now
-			if events, err := Query(c.ctx, lgi, since, time.Now()); err == nil {
-				// println("found logs:", len(events))
+			if events, err := Query(c.ctx, lgi, since, time.Now()); err != nil {
```

Comment:
this is still a no-op unless there's side effects for the Query call.

Reply:
the `else` is still there.